src/Razor/benchmarks/Microsoft.AspNetCore.Razor.Performance/MSN.cshtml

```diff
@@ -1980,9 +1980,9 @@ alt="Mostly Cloudy" title="Mostly Cloudy" data-icon="3" />
  </ul>
  <ul class="tertiary"  data-aop="tertiary"  data-id="249" data-m='{"i":249,"p":242,"n":"tertiary","y":8,"o":3}'>
  <li  data-id="250" data-m='{"i":250,"p":249,"n":"HeadlineItemViewModel","y":13,"l":"BBn8rQE","h":0,"v":"news","c":"newsworld","o":1}'  >
-<a href="/en-us/news/world/chief-suspect-is-an-isis-militant-whose-own-family-wished-him-dead/ar-BBn8rQE?li=AA54ur"  >
+<a href="/en-us/news/world/chief-suspect-is-a-militant-whose-own-family-wished-him-dead/ar-BBn8rQE?li=AA54ur"  >
 <img alt="Abdelhamid Abaaoud" data-src="{&quot;default&quot;:&quot;//img-s-msn-com.akamaized.net/tenant/amp/entityid/BBn8rQC.img?h=75&amp;w=100&amp;m=6&amp;q=60&amp;u=t&amp;o=t&amp;l=f&amp;x=229&amp;y=204&quot;}" src="//static-hp-wus.s-msn.com/sc/9b/e151e5.gif" title="via Reuters" /> <div>
-<h4 >Chief Suspect Is an ISIS Militant Whose Own Family Wished Him Dead</h4>
+<h4 >Chief Suspect Is an People Militant Whose Own Family Wished Him Dead</h4>
```

Comment:
nit: "a Person" 😃

Reply:
lol

Reply:
I removed myself from the build team so feel free tomerge when you  feel it's fit